FRPs/FRP-6.md

```diff
@@ -0,0 +1,53 @@
+# FRP-6: Literature review of auction mechanism that fit in with Flashbots.
+
+
+**MEV Fellow:** @sbaks0820
+</br> **Status:** Draft
+</br> **Updated:** 24.11.2020
+
+### Abstract
+The puspose of this isssue is to investigate and survey the existing literature of auction mechanism design to motivate the design of the block space and transaction ordering auctions enabled by MEV-GETH.
+An MEV-GETH auction is different from a traditional auction in that users submit and bid on tx sequences that are, potentially, non-overlapping.
+The current implementation of MEV-GETH accepts only a single transacton sequence into a block, however, a more sophisticated auction mechanism might permit multiple "bundles" with interaction between winning, non-overlapping bundles.
+Therefore careful consideration is required to design a good auction mechanism for MEV-GETH.
+In addition to surveying existing literature, it is impotant to survey auction designs in existing crypto projects so understand the practical requirements of a decentralized auction such as privacy, security, auditability, and efficiency.
+
+### Defining the Research Question
+
+The high-level research question is: What is a good auction mechanism for MEV-GETH auctions? 
+
+The first step in answering this broad question is first surveying and evaluating the existing literature and practice of auction mechanism design in a decentralized setting.
+In order to tackle this objects is to answer the following questions:
+* In what ways do crypto auctions differ from existing literature: what are the impact of easy collusion forming on cooperative strategies and does current literature fall short of addressing this concern? Ad auctions have a similar setting, perhaps a better survey of how they deal with it? (see Methodolody for survey of existing projects, i.e. industry anecdotes). 
+* What assumptions can be made about the participants and their capabilities (includes a potentially malicious miner)?
```

Comment:
One potential thing you might want to do here is to classify the strategy space, akin to what algorithmic game theorists do in the picture below.

I will note that the mining auction case is quite different, however, because 
a) it is an iterated, memory-filled game 
b) correlations between auctions can be long and quite persistent due to hedging mechanisms (e.g. gas tokens)
c) you have to decide on a notion of 'approximate equilibrium' that you are satisfied with

Part c) is quite delicate --- for instance, do you want approximations in the sense of eps-Nash? Do you want it in a slightly stronger sense of, "there exists a polynomial sequence of approximations A_i, such that the limit $A^* = lim_{i \rightarrow \infty} A_i$ is the exact optimal strategy"? These are subtly different (c.f. the proof of the Nash embedding theorem) but can give you wildly different auction behaviors.

![image](https://user-images.githubusercontent.com/37755040/102914673-a87f8780-444e-11eb-8711-321f265fe53f.png)

Reply:
For part c, is there a way for us to punt the problem of choosing which equilibrium to searchers who construct the most optimized blocks? The competition within searchers would then be in which equilibria they optimize for and the strategies to reach those equilbrium?